### PR TITLE
Friends page

### PR DESCRIPTION
**Pull request by Graves067**
Friends page formatting is completed, at current there are 10 example pages. Furthermore, we will decide if we want to make the Side bar an universal asset for the project. Minor adjustments needed to layout.

### CODE REVIEW

File: flask/templates/Friendspage.html [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+    <head> 
+
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1 ">
+<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-iYQeCzEYFbKjA/T2uDLTpkwGzCiq6soy8tYaI1GyVh/UjpbCx/TYkiZhlZB6+fzT" crossorigin="anonymous">
+<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/js/bootstrap.bundle.min.js" integrity="sha384-OERcA2EqjJCMA+/3y+gxIOqMEjwtxJY7qPCqsdltbNJuaOe923+mo//f6V8Qbsw3" crossorigin="anonymous"></script>
+<script src="https://kit.fontawesome.com/80c51aa779.js" crossorigin="anonymous"></script>
+
+</head>
+
```
Comment (jacobtizel) [inline flask/templates/Friendspage.html:12]: Try to take out the head stuff (bootstrap, fontawesome, etc) to the base.html file and use the {% extend %} thing to get everything from base.html in the friends page.

Comment (jacobtizel) [reply flask/templates/Friendspage.html:12]: check to see if this breaks the other pages though.

File: flask/main.py
Comment (jacobtizel) [inline flask/main.py:1]: This file is deprecated and will be removed with a git pull origin main.

File: project/templates/Friendspage.html [hunk 2]
```diff
@@ -0,0 +1,258 @@
+
+<html>
+    <head> 
+
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1 ">
+<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-iYQeCzEYFbKjA/T2uDLTpkwGzCiq6soy8tYaI1GyVh/UjpbCx/TYkiZhlZB6+fzT" crossorigin="anonymous">
```
Comment (jacobtizel) [inline project/templates/Friendspage.html:7]: Try to take out the head stuff (bootstrap, fontawesome, etc) to the base.html file and use the {% extend %} thing to get everything from base.html in the friends page. Check to see if it breaks the other pages though.

**Review by jacobtizel — APPROVED** [review]
Looks great! Just pull out Bootstrap and Fontawesome to the base.html file so we can use it with everything else.

**Review by Sawneet-24 — APPROVED** [review]
Everything looks great. Well done

**Pull request merged**